tests/Integration/Behaviour/Features/Scenario/Order/order_multi_invoices.feature

```diff
@@ -0,0 +1,698 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags order-multi-invoices
+@reset-database-before-feature
+@order-multi-invoices
+@clear-cache-before-feature
+Feature: Order from Back Office (BO)
+  In order to manage orders for FO customers
+  As a BO user
+  I need to be able to customize orders from the BO
+
+  Background:
+    Given email sending is disabled
+    And the current currency is "USD"
+    And country "US" is enabled
```

Comment:
```suggestion
    And the country "US" is enabled
```

Reply:
I fixed the other suggestion for behat sentences, but this one is quite old, used in many different features with unrelated domains So I prefer not to fix it here to avoid making the PR too big and having potential conflicts with other pending PRs

Reply:
npb